### DISCUSSION TITLE
A new home for Lookbook

### DISCUSSION THREAD

**Discussion by allmarkedup** [lookbook-hq/lookbook#615] [Announcements]
As some of you may have already spotted, Lookbook has just been moved out from under the wings of [ViewComponent](https://github.com/ViewComponent) and into it's own shiny new Github org at [github.com/lookbook-hq](https://github.com/lookbook-hq) 😎

Lookbook was originally conceived as a simple, drop-in replacement UI to provide a better experience for browsing and interacting with [ViewComponent previews](https://viewcomponent.org/guide/previews.html). It was heavily tied into ViewComponent and so it made perfect sense for the project to live under the VC umbrella and I was honoured when @joelhawksley invited me in.

However one of the main aims of the v2.0 release was to start the process of decoupling Lookbook from ViewComponent so that it could serve as a more general-purpose preview tool, with support for frameworks such as [Phlex](https://phlex.fun) as well as 'vanilla' Rails partials and templates. And the intention for future versions of Lookbook is to continue widening that net by adding support for more preview types (ActionMailer previews?) as well as more features to help developers build and test Ruby components with confidence and in isolation from their main application.

With this in mind it felt like moving Lookbook into it's own organisation was the next logical step in the transition away from its ViewComponent-focussed roots and towards the rather lofty goal of making Lookbook the de-facto **UI development environment** and **documentation tool** for Rails apps, regardless of frontend technology choice.

Lots to do! 🙂

**Comment by Jackson200** [comment]
ActionMailer Previews 👍  yay!
